src/Network/Response.php

```diff
@@ -56,7 +56,7 @@ class Response
         303 => 'See Other',
         304 => 'Not Modified',
         305 => 'Use Proxy',
-        306 => 'Switch Proxy',
+        306 => '(Unused)',
```

Comment:
It has - as far as I can understand - been deprecated by the IETF - see my original https://github.com/cakephp/cakephp/pull/9528/files#diff-a57bc7407c64664db661cb46c465f363R85 especially https://tools.ietf.org/html/rfc2616#section-10.3.7